matrix-sdk/src/main/java/org/matrix/androidsdk/data/cryptostore/db/CryptoFileStoreImporter.kt

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2018 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.androidsdk.data.cryptostore.db
+
+import android.content.Context
+import io.realm.Realm
+import io.realm.RealmList
+import io.realm.kotlin.where
+import org.matrix.androidsdk.data.cryptostore.MXFileCryptoStore
+import org.matrix.androidsdk.data.cryptostore.db.model.*
+import org.matrix.androidsdk.data.cryptostore.db.query.getOrCreate
+import org.matrix.androidsdk.rest.model.login.Credentials
+import org.matrix.androidsdk.util.Log
+
+/**
+ * This class migrate the legacy FileCryptoStore to the Realm DB
+ * TODO Try this several times? depending on crash, etc...
+ * -> initial Data will not be called...
```

Comment:
It was just some ideas, but I've updated the comment.
Actually, if the migration fails, I'm not sure this code will be executed again, because the database will be initialized.